tests/e2e/utils/performance.ts

```diff
@@ -83,11 +89,12 @@ export const recreatePerformanceFile = () => {
  * @param {Object} metrics An object containing multiple trials' data.
  * @return {Object} The averaged results.
  */
-export const averageMetrics = ( metrics ) => {
+export const averageMetrics = ( metrics: Record< string, number[] > ) => {
 	const results = {};
 	for ( const [ key, value ] of Object.entries( metrics ) ) {
 		results[ key ] =
-			value.reduce( ( prev, curr ) => prev + curr ) / NUMBER_OF_TRIALS;
+			value.reduce( ( prev, curr ) => prev + curr ) /
+			performanceNumberOfTrials;
 	}
 	return results;
 };
```

Comment:
Can we add the return type in the function? Like this?


```suggestion
export const averageMetrics = (
	metrics: Record< string, number[] >
): Record< string, unknown > => {
	const results = {};
	for ( const [ key, value ] of Object.entries( metrics ) ) {
		results[ key ] =
			value.reduce( ( prev, curr ) => prev + curr ) /
			performanceNumberOfTrials;
	}
	return results;
};
```

Reply:
Sure! I added two new types (`Metrics` and `AverageMetrics`) to that file to help with that on [58c4adf](https://github.com/Automattic/woocommerce-payments/pull/10352/commits/58c4adfad326d2e05d15d820d21d7aa9f5d06892). Thanks for raising it!